rllib/policy/tf_policy.py

```diff
@@ -983,35 +983,47 @@ class EntropyCoeffSchedule:
 
     @DeveloperAPI
     def __init__(self, entropy_coeff, entropy_coeff_schedule):
-        self.entropy_coeff = get_variable(
-            entropy_coeff,
-            framework="tf",
-            tf_name="entropy_coeff",
-            trainable=False)
-
+        self._entropy_coeff_schedule = None
```

Comment:
Should you add a test for this?

Reply:
You mean for the mixin class? Also for `LearningRateSchedule`? Sure.
We do have extensive tests for the different `Schedule` classes.

Reply:
done

Reply:
Added extensive schedule testing (within a Policy) to test_ppo.py. These will test tf/tf2/torch policies for their handling of different lr- and entropy-coeff schedules.